BitwardenShared/Core/Vault/Services/API/Fixtures/syncWithAccountKeysV2Profile.json

```diff
@@ -0,0 +1,44 @@
+{
+  "profile": {
+    "id": "c8aa1e36-4427-11ee-be56-0242ac120002",
+    "name": null,
+    "email": "user@bitwarden.com",
+    "emailVerified": false,
+    "premium": false,
+    "premiumFromOrganization": false,
+    "masterPasswordHint": null,
+    "culture": "en-US",
+    "twoFactorEnabled": false,
+    "key": "key",
+    "privateKey": "private key",
+    "accountKeys": {
+      "publicKeyEncryptionKeyPair": {
+        "wrappedPrivateKey": "WRAPPED_PRIVATE_KEY",
+        "publicKey": [1, 2, 3, 4, 5],
```

Comment:
Question / note: This is a base64 encoded string on the server https://github.com/bitwarden/server/blob/8a39481fec87848e19cdd92d620d3e23b97901cb/src/Api/KeyManagement/Models/Requests/PublicKeyEncryptionKeyPairRequestModel.cs#L9. Is it intended to be an array of bytes here?

Reply:
Nice catch! You're right, I saw an object in the clients repo with that type and I mistakenly applied it here directly missing it was being converted from a string in the response. Fixed and removed the `UnsignedPublicKey` typealias as it's no longer use.